src/Coders/Model/Model.php

```diff
@@ -745,7 +745,7 @@ public function getConnectionName()
      */
     public function hasCustomPrimaryKey()
     {
-        return count($this->primaryKeys->columns) == 1 &&
+        return ($this->primaryKeys->columns) == 1 &&
```

Comment:
What does this supposedly do?

Reply:
I had this same problem, an Exceptino was thrown: `ErrorException  : count(): Parameter must be an array or an object that implements Countable`. Turns out there wasn't a primary key defined in one of my tables...

Reply:
That's helpful. Thanks!

Reply:
how about check if the variable of $this->primaryKeys->columns is array?